packages/components/src/components/SquigglePlayground/index.tsx

```diff
@@ -105,6 +112,19 @@ export const SquigglePlayground: React.FC<PlaygroundProps> = (props) => {
     isRunning: boolean;
   }>({ output: undefined, isRunning: false });
 
+  useEffect(() => {
+    const _output = output.output?.output;
+    if (_output && _output.ok) {
+      const exports = _output.value.exports;
+      const _exports: ModelExport[] = exports.entries().map((e) => {
+        return { variableName: e[0], title: e[1].title() };
+      });
+      onExportsChange && onExportsChange(_exports);
+    } else {
+      onExportsChange && onExportsChange([]);
+    }
+  }, [output, onExportsChange]);
```

Comment:
On challenge with ``onOutputChange`` are the types. I'm assuming this would pass in ``SquiggleOutput``, but SquiggleOutput has a lot of complexity. 

This becomes a problem with ``VersionedSquigglePlayground``.

Reply:
Here's the type for SquiggleOutput. Do you think we should pass this in, to VersionSquigglePlayground?

```
export type SquiggleOutput = {
  output: result<
    {
      exports: SqDict;
      result: SqValue;
      bindings: SqDict;
    },
    SqError
  >;
  code: string;
  executionId: number;
  executionTime: number;
};
```

Reply:
`SquiggleOutput` is complicated / too components-specific, but `SquiggleOutput['output']`, which is the same as `ReturnType<SqProject["getOutput"]>`, is a good middle ground.

It's slightly less flexible because it's harder to associate the output with the code that it came from or get execution metadata, but it should be good enough.

(And it would still be possible to extract code, through `output.context.source`)

Reply:
By ``SquiggleOutput['output']``, do you mean this part?
```
 output: result<
    {
      exports: SqDict;
      result: SqValue;
      bindings: SqDict;
    },
    SqError
  >
```

That part is the main complex part. I'm nervous that SqDict and SqValue will change between versions, which I would assume would mess with VersionedPlayground.

Reply:
Huh, wait, I'm not sure how to do that type either because you still have to choose which exactly `SqProject` / `SqValue` type to expose from `VersionedSquigglePlayground`, and they might be different in different releases.

We could copy-paste types and hope that they don't change too much, but we'd lose type safety guarantees.

I think I have a better idea on how to make versioned-playground more type-safe, but I need to make an experiment to check if it's viable.

Reply:
So what do you want me to do now? The nice thing about the ``modelExports`` was that they were very simple at least, but could still have some issues now. 

I could wait for other stories, and just leave this for now.